From 345d1e7645acf26a4aa51d7e77cc29cd1fe2e1a6 Mon Sep 17 00:00:00 2001
From: Hu Ziji <huziji@marvell.com>
Date: Mon, 1 Aug 2016 10:42:06 -0700
Subject: [PATCH 0678/2241] mmc: sdhci-xenon: Add support to Xenon SDHC in
 mmc/host/Makefile

Enable compiling xenon-sdhci.c and xenon-sdhci-phy.c

Change-Id: Ie3e7b10f89f7dd3f006f0aaad5bbb3410e433566
Signed-off-by: Hu Ziji <huziji@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/31898
Reviewed-by: Omri Itach <omrii@marvell.com>
Verified-Armada37x0: Kostya Porotchkin <kostap@marvell.com>
Verified-Armada7K: Kostya Porotchkin <kostap@marvell.com>
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 drivers/mmc/host/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/mmc/host/Makefile b/drivers/mmc/host/Makefile
index 3595f83..749e925 100644
--- a/drivers/mmc/host/Makefile
+++ b/drivers/mmc/host/Makefile
@@ -79,3 +79,5 @@ obj-$(CONFIG_MMC_SDHCI_ST)		+= sdhci-st.o
 ifeq ($(CONFIG_CB710_DEBUG),y)
 	CFLAGS-cb710-mmc	+= -DDEBUG
 endif
+
+obj-$(CONFIG_MMC_SDHCI_XENON)		+= sdhci-xenon.o sdhci-xenon-phy.o
-- 
2.7.4

